docs/contribute/saros-testing-framework.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Saros Testing Framework (STF)
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+For further information
+about STF take a look at the [STF User
+Manual](http://saros-build.imp.fu-berlin.de/stf/STF_Manual.pdf).
```

Comment:
This is the latest version, although it is outdated (some API methods were added) it is still useful.

Reply:
Concurrency problems while reviewing ;) This makes my question obsolete.

Reply:
@srossbach I will add your comment to the documentation

Reply:
Done